tools/binlog/tidb-binlog.md

```diff
@@ -0,0 +1,287 @@
+---
```

Comment:
it's oldest binlog version, can we delete it or rename it to `tidb-binlog-local`

Reply:
rename to tidb-binlog-local i think will be better. 
update in https://github.com/pingcap/docs-cn/pull/1264/commits/d532c6e6675c82fb1bfc83b93d394e8bf2d8e158